docs/static/security/es-security.asciidoc

```diff
@@ -2,11 +2,15 @@
 [[es-security-on]]
 == {es} security on by default
 
-{es} {ref}/configuring-stack-security.html[security is on by default] starting in 8.0.
+Security is on by default starting in 8.0.
 {es} generates its own default self-signed Secure Sockets Layer (SSL) certificates at startup. 
 
-For an on-premise {es} cluster, {ls} must establish a secure connection using the self-signed SSL certificate before it can transfer data.  
-On the other hand, {ess} uses standard publicly trusted certificates, and therefore setting a cacert is not necessary.
+{ls} must establish a Secure Sockets Layer (SSL) connection before it can transfer data to an {es} cluster. 
```

Comment:
```suggestion
{ls} must establish a Secure Sockets Layer (SSL) connection before it can transfer data to a secured {es} cluster. 
```

It is on by default, but can be disabled. Adding a qualifying _"secured"_ maybe helpful.